Core/clim-basic/pattern.lisp

```diff
@@ -388,7 +388,10 @@ Returns a pattern representing this file."
          (transformation (transformed-design-transformation effective-pattern))
          (inv-tr (invert-transformation transformation)))
     (multiple-value-bind (x y) (transform-position inv-tr x y)
-      (design-ink source-pattern (round x) (round y)))))
+      ;; It is important to not use ROUND here, since when the
+      ;; fractional part is exactly 0.5, we get wrong dimensions.
+      ;; loke 2019-01-06
```

Comment:
make it `-- loke 2019-01-06` for consistency with other comments. it may be in the same line as the comment.